test/functional/python/CMakeLists.txt

```diff
@@ -7,4 +7,5 @@ add_test(NAME pytests
 )
 set_property(TEST pytests APPEND PROPERTY
 		ENVIRONMENT PYTHONPATH=${Spglib_Python_BINARY_DIR}
+		            PATH=${CMAKE_INSTALL_PREFIX}/lib:$ENV{PATH}
```

Comment:
These tests should be run within the build directory without requiring install in order to support IDE debugging and rapid development

Reply:
Yes, I know, but step by step ....